lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +75,23 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict) or not isinstance(pair, tuple):
+        return None
+    dictionary = {}
+    united = pair[0]+pair[1]
```

Comment:
concatination of the string? Use fstring

Reply:
fixed